common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -3898,6 +3898,9 @@ schematic = (function() {
 	    tool.style.verticalAlign = 'middle';
 	    tool.style.cursor = 'default';
 
+        // add alt using tip
+        tool.setAttribute('alt', tip);
```

Comment:
@clrux This also adds an alt attribute to the ```<span>``` elements that are used for the DC and TRANS controls.  Unfortunately, those are currently very inaccessible (no keyboard support).  We need to make those buttons or aria-controls.  Leave it up to you if we should address that here or in a separate ticket.  I vote for creating a second ticket because this content type has many many issues.  It's nice to just get this alt text one out of the way. I don't want to invest too much time and effort on this assessment type right now.